Automattic/woocommerce-payments
# Meetup: E2E - Add accept_tos=1 when adding linking account

Fixes the issue that constantly asks for accepting ToS after setting up E2E server site.
More context p1625187109034600/1624940773.016100-slack-C025WLAGYBZ

Actually, this PR only works if the related server PR 945 is merged too. 

#### Changes proposed in this Pull Request

See the server PR. 

#### Testing instructions

- Pull the branch in the server PR in the this folder: `tests/e2e/deps/wcp-server/`.
- Run `npm run test:e2e-setup`
- Check the E2E client site 
- Expectation: no longer see the modal asking for accepting ToS when visiting any page under WP Admin > Payments. 





-------------------

Hmm, I am not sure if these checklist items below are relevant for this issue? Feel free to let me know. This is my very first PR.
